src/pages/billing/Project.js

```diff
@@ -226,8 +228,6 @@ const ProjectDetail = ({ authorizeAndLoadAccounts, billingAccounts, billingProje
     { namespace: billingProject.projectName },
     _.map('workspace', workspaces)
   ), [billingProject, workspaces])
```

Comment:
nit: re-add spacer line
```suggestion
  ), [billingProject, workspaces])

```

Reply:
Will put this in my next PR to avoid merge issues.